ai-ml/jark-stack/terraform/src/service/ray-service.yaml

```diff
@@ -16,6 +16,7 @@ spec:
     importPath: dogbooth:entrypoint
     runtimeEnv: |
       env_vars: {"MODEL_ID": "askulkarni2/dogbooth"}
+      #env_vars: {"MODEL_ID": "stabilityai/stable-diffusion-2-1"}
```

Comment:
Did you mean to use both models? This looks like commented

Reply:
No I left there as an example for using upstream model if someone wants to test it.